apps/web-giddh/src/app/settings/integration/setting.integration.component.ts

```diff
@@ -768,11 +770,11 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
                     userAmountRanges.controls[index].get('maxBankLimit').patchValue('custom');
                     userAmountRanges.controls[index].get('amount').patchValue(null);
                     userAmountRanges.controls[index].get('amount').setErrors({ 'incorrect': true });
-                    userAmountRanges.controls[index].get('amount').setValidators(Validators.compose([Validators.required, Validators.maxLength(this.maxLimit)]));
+                    userAmountRanges.controls[index].get('amount').setValidators(Validators.compose([Validators.required]));
                     this.toasty.infoToast('You can not select max bank limit more than 1');
                 }
             } else {
-                userAmountRanges.controls[index].get('amount').setValidators(Validators.compose([Validators.required, Validators.maxLength(this.maxLimit)]));
+                userAmountRanges.controls[index].get('amount').setValidators(Validators.compose([Validators.required]));
```

Comment:
Check with backend team what is the max length of interger they support as error will be thrown since we are not limiting at UI

Reply:
Yes @prateek-giddh we have updated this and it's working